src/components/Ticker/Ticker.js

```diff
@@ -144,22 +143,15 @@ const Ticker = ({ images, arrowLeft, arrowRight }) => {
   const stop = () => {
     isRunning.current = false;
   };
-
   return (
     <div ref={container} className={styles.container}>
       <a.ul
         ref={ticker}
         style={{ transform: props.x.interpolate(iTranslate) }}
         className={styles.ticker}
       >
-        {items.map(({ childImageSharp }, index) => {
-          return (
-            <li key={index} className={styles.item}>
-              <div className={styles.card}>
-                <Img fluid={childImageSharp.fluid} draggable={false} />
-              </div>
-            </li>
-          );
+        {items.map(({ name, element }, index) => {
+          return <Fragment key={name + index}>{element}</Fragment>;
```

Comment:
this should be good enough
```suggestion
          return <Fragment key={name}>{element}</Fragment>;
```

Reply:
Without index we don't have unique key, because we copy incoming array

Reply:
ok, then we need to add a number of the copy

like that
```
1.jpg1
2.jpg1
3.jpg1
1.jpg2
2.jpg2
3.jpg2
1.jpg3
2.jpg3
3.jpg3
```